src/modules/movies.js

```diff
@@ -28,33 +28,36 @@ export default function reducer(state: Object = initialState, action: Object) {
   }
 }
 
-export function receiveMovies(json: Array<number | string> = []) {
+export function receiveMovies(json: Object) {
   return { type: MOVIE_LOADED, data: json, isLoaded: true };
 }
 
 export function handleErrors(error: Object) {
   return { type: MOVIE_ERROR, data: error, isLoaded: false };
 }
 
-export function movies(value: string | number) {
+export function movies(value: string | number, page: number) {
   if (value) {

```

Comment:
For such cases, better make two functions and then call them separately. 

In what cases we can call this function and not pass `value` to it?

Reply:
We call this function without `value` when we load main page on first page load, as default.